doc/examples/reconst_cti.py

```diff
@@ -0,0 +1,152 @@
+"""
+========================================================================
+Reconstruction of the diffusion signal with the correlation tensor model
+========================================================================
+
+The Correlation Tensor Model (CTI) is an advanced extension of the Diffusion Kurtosis Model (see :ref:sphx_glr_examples_built_reconstruction_reconst_dki.py). [ not sure]
+In addition to the DKI, CTI introduces a more complete representation of the underlying microstructural complexity. Specifically, the CTI differentiates between different sources of kurtosis by distinguishing between isotropic, anisotropic, and microscopic contributions. This refined characterization provides richer insights into tissue microstructure and can reveal features that may be obscured or conflated in simpler models.
```

Comment:
Write instead: Therefore, in addition to the kurtosis sources related to diffusion variance across compartments (K_aniso and K_iso related to the microscopic anisotropy and the variance of compartments mean diffusivities respectively), CTI also measures K_micro which quantifies non-Gaussian diffusion effects deviating for the multiple Gaussian component tissue representation (effects like restricted diffusion, exchange, compartments structural disorder like cross-sectional variance).

Reply:
Here cite the work by Lisa Novello and Rita Alves:

Novello, L., Henriques, R.N., Ianus, A., Feiweier, T., Shemesh, N., Jovicich, J. 2022. 
In vivo Correlation Tensor MRI reveals microscopic kurtosis in the human brain on 
a clinical 3T scanner. Neuroimage 254: 119137 doi: 10.1016/j.neuroimage.2022.119137 

 Alves, R., Henriques, R.N., Kerkelä, L., Chavarrías C., Jespersen, S.N., Shemesh, N. 
2022. Correlation Tensor MRI deciphers underlying kurtosis sources in stroke, 
Neuroimage 247: 118833. doi: 10.1016/j.neuroimage.2021.118833